### ISSUE TITLE
Move between folders?

### ISSUE THREAD

**Issue by cgb** [ryanblenis/MeshCentral-ScriptTask#6]
Hey there,

Is moving scripts between foldres supported?  I can see references to dropMove in the code but when I try, the drag behaviour detects mouse over of another folder and shows the little arrow, but doesn't seem to move it.  I've tried in Chrome & Firefox but doesn't seem to work.  Am I doing something wrong? :)

Thanks!

**Comment by ryanblenis** [comment]
Yes it is. I just tested with the latest Firefox and Chrome, and it works as expected. What server version / browser version are you using that it does not move? Is there any error?

Thanks

**Comment by cgb** [comment]
Thanks Ryan:

- ScriptTask: 0.0.15
- Meshcentral: 0.5.47
- Firefox 76.0
- Linux OS (Fedora 30)

No error shown.  Any suggestions on debugging?

**Comment by cgb** [comment]
Ok, I have some more info.  I was tested with a Windows OS & latest firefox and managed to successfully move a script.  I found the mouse over icon changes to an arrow when you are on top of anywhere on the folder name, but the move only works if you are pointing at the folder icon.

So I've worked out how to do it, but I wonder if it can move successfully when you drop the script on any part of the icon/folder name collectively?
